Java/cp/RemoveDuplicateElement.java

```diff
@@ -1,9 +1,12 @@
+// import the Scanner class
```

Comment:
No need to add this comment, instead add the description of the problem here.

Reply:
Okay, I will remove the comment. The description of the problem is already present.
https://github.com/TesseractCoding/NeoAlgo/pull/803/files#diff-0c8550122b2ec1ce669256c8a245d914L3